docs/source/providers.rst

```diff
@@ -261,6 +273,8 @@ Account Methods
 
 .. autofunction:: getRegistry
 
+.. autofunction:: getNewTesseractsByAccountFilter
```

Comment:
can be moved under query methods.

Reply:
6144a3467ccfbaf92c4f59eeb12fffd747006319